blockchain/projects_test.go

```diff
@@ -620,16 +504,16 @@ func Test_DeployContract(t *testing.T) {
 				}
 			}`
 
-		accounts, err := projects.CreateInitialAccounts(proj.ID)
+		_, err := projects.CreateInitialAccounts(proj.ID)
 		assert.NoError(t, err)
 
-		accA, err := projects.DeployContract(proj.ID, accounts[0].Address, scriptA)
+		deploy1, err := projects.DeployContract(proj.ID, model.NewAddressFromString("0x05"), scriptA)
 		require.NoError(t, err)
-		assert.Equal(t, accA.DeployedCode, scriptA)
+		assert.Equal(t, deploy1.Title, "HelloWorldA")
 
-		accB, err := projects.DeployContract(proj.ID, accounts[1].Address, scriptB)
+		deploy2, err := projects.DeployContract(proj.ID, model.NewAddressFromString("0x06"), scriptB)
 		require.NoError(t, err)
-		assert.Equal(t, accB.DeployedCode, scriptB)
+		assert.Equal(t, deploy2.Title, "HelloWorldB")
```

Comment:
order of all asserts should be swapped: it is `expected value, actual value`. This is really annoying order to me as well, but it's good we stick to the correct API otherwise output is confusing.

Reply:
Fixed (at least everywhere I could see at first glance)